js/navbar.js

```diff
@@ -1,3 +1,16 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+  const checkURL = window.location.href;
+  const productionURL = "https://welcome.realdevsquad.com/";
+  let userLogin = document.getElementById('user-login');
+
+  if (checkURL === productionURL) {
```

Comment:
Suggestion:
```js
  const clientId = (checkURL === productionURL ? '23c78f66ab7964e5ef97' : 'c4a84431feaf604e89d1';
  userLoginEl.href = `https://github.com/login/oauth/authorize?client_id=${clientId}`;
```

Reply:
Thanks @ankushdharkar . @dhanush027  please make the required changes.